internal/resolution/pm/gradle/job.go

```diff
@@ -53,22 +61,25 @@ func (j *Job) Run() {
 
 	if err != nil {
 		if permissionErr != nil {
-			j.Errors().Critical(util.NewPMJobError(permissionErr.Error()))
+			j.handleError(util.NewPMJobError(permissionErr.Error()))
 		}
-		j.Errors().Critical(util.NewPMJobError(err.Error()))
+		j.handleError(util.NewPMJobError(err.Error()))
 
 		return
 	}
 
-	j.SendStatus("creating dependency graph")
+	status := "creating dependency graph"
+	j.SendStatus(status)
 	_, err = dependenciesCmd.Output()
 
 	if permissionErr != nil {
-		j.Errors().Warning(util.NewPMJobError(permissionErr.Error()))
+		cmdErr := util.NewPMJobError(permissionErr.Error())
+		cmdErr.SetIsCritical(false)
+		j.handleError(cmdErr)
```

Comment:
Could we add the run command as well?
`cmdErr.SetCommand(dependenciesCmd.String())`